README.md

```diff
@@ -109,7 +127,7 @@ Supported algorithms "RSA" or "ECDSA".
 Signs a metadata.xml file by using a smartcard (such as a yubikey). 
 Supported algorithms "RSA" or "ECDSA".
 
-    ./build/install/verify-eidas-trust-anchor/bin/verify-eidas-trust-anchor connector-metadata sign-with-smartcard \
+    connector-metadata sign-with-smartcard \
```

Comment:
```suggestion
    trust-anchor connector-metadata sign-with-smartcard \
```

Reply:
So no because in the docker version you don't need `trust-anchor`, because that's implied by the ENTRYPOINT in the Dockerfile. Which I agree is confusing, I've kinda attemped to explain above but maybe failed rather :/

Reply:
Okay, it looked like trust-anchor was the command.  Maybe if you put something like `$COMMAND` before each of them you wouldn't need to explain anything?